.github/workflows/lint.yml

```diff
@@ -0,0 +1,50 @@
+name: Lint
+
+on: push
+
+jobs:
+  run-frontend-linters:
+    name: Run frontend linters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      # ESLint and Prettier must be in `package.json`
```

Comment:
Should `prettier` be added as a dep? It's missing from `package.json`.

Reply:
I honestly forget why these comments are there. I don't think it's necessary, but I've added `prettier` back in just to be sure. 👍